gomint-api/src/main/java/io/gomint/world/block/BlockElement.java

```diff
@@ -0,0 +1,18 @@
+package io.gomint.world.block;
+
+/**
+ * @author Kaooot
+ * @version 1.0
+ */
+public interface BlockElement extends Block {
+
+    int getProtons();
+
+    int getElectrons();
+
+    int[] getNeutrons();
```

Comment:
Please add a documentation why a element has multiple neutron values and what todo with it